framework/test/jest/integration/specs/modules/chain/blocks/chain/apply_block.spec.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+describe('integration test (blocks) - chain/applyBlock', () => {
+	describe('applyBlock', () => {
+		describe('applyConfirmedStep', () => {
+			describe('after applying new block fails', () => {
+				it.todo('should have pooled transactions in queued state');
```

Comment:
Maybe combine this in an `it` with the `describe` above it?

Reply:
There are multiple specs in this describe, so seems ok to not merge.